docs/RELEASE-NOTES.rst

```diff
@@ -7,6 +7,25 @@ Release Notes for BIG-IP Controller for Cloud Foundry
 Added Functionality
 ```````````````````
 
+v1.1.0
+------
+
+Added Functionality
+```````````````````
+* L7 (HTTP) per route configuration through Route Services when controller is run as a Service Broker
+
+  * Virtual Server for L7 (HTTP) route can have its own Policies, Profiles and SSL Profiles
+  * Pool for L7 (HTTP) route can have its own load balancing mode and Health Monitors
+
+* Virtual Server targeting Virtual Server architecture
+
+Limitations
+```````````
+* Only support Route Service bindings
```

Comment:
Remove "only". Explicitly state what we don't support, instead of saying we only support one thing.
```
* Controller cannot accept Arbitrary Parameters via `cf create-service|bind-service -c`
* Controller doesn't support the use of `cf update-service`.
```